spring-cloud-sleuth-core/pom.xml

```diff
@@ -76,6 +76,11 @@
 			<optional>true</optional>
 		</dependency>
 		<dependency>
+			<groupId>org.springframework.data</groupId>
+			<artifactId>spring-data-rest-webmvc</artifactId>
+			<optional>true</optional>
```

Comment:
style, but usually I comment what class or purpose optional deps are needed for, when adding. Helps with any future cleanup or questions.